projects/canopy/src/lib/primary-message/primary-message-title/primary-message-title.component.html

```diff
@@ -0,0 +1 @@
+<ng-content></ng-content>
```

Comment:
Given this is only used to display a title I wonder whether the content should be wrapped in an lg-heading to force the correct semantics

Reply:
I thought about this while working on it and I don't believe the content of the title is actually a heading - just styled like one.
To me the title is just a simple message (like a validation or error message); "Unfortunately something went wrong" or "There are no transactions to filter" and "All done" shouldn't be headings but just simple text that is read out when the message is triggered (that's why this component had a default role of alert).

In case of the component being used in a confirmation page, the title might be a heading and it's left to the user to decide that.

Reply:
ok that makes sense :-)